src/materialized/src/bin/materialized/main.rs

```diff
@@ -481,16 +481,16 @@ fn run(args: Args) -> Result<(), anyhow::Error> {
     let metrics_registry = MetricsRegistry::new();
     // Configure tracing.
     {
-        use tracing_subscriber::filter::{EnvFilter, LevelFilter};
+        use tracing_subscriber::filter::{LevelFilter, Targets};
         use tracing_subscriber::fmt;
         use tracing_subscriber::layer::{Layer, SubscriberExt};
         use tracing_subscriber::util::SubscriberInitExt;
 
-        let env_filter = EnvFilter::try_new(args.log_filter)
+        let filter = Targets::from_str(&args.log_filter)
             .context("parsing --log-filter option")?
             // Ensure panics are logged, even if the user has specified
             // otherwise.
-            .add_directive("panic=error".parse().unwrap());
+            .with_target("panic", LevelFilter::ERROR);
```

Comment:
this will no longer apply to the ThirdPartyMetric subscriber, in case that that matters to you

also, is `panic` a valid "target prefix"? https://docs.rs/tracing-subscriber/0.3.3/tracing_subscriber/filter/targets/struct.Targets.html#method.with_target
it looks like its just the module path?

Reply:
Good question! The answer is that it is valid... for us. We explicitly log panics with the `panic` target: https://github.com/MaterializeInc/materialize/blob/7d889cb9641d73f95716a02f0581b221db612a8c/src/materialized/src/bin/materialized/main.rs#L827